include/convex_bodies/LMI.h

```diff
@@ -0,0 +1,180 @@
+//
```

Comment:
Since we will need to use spectrahedron with different kinds of matrices (dense, sparse), I wanted to separate the actual class Spectrahedron from the required algebra. So, I imagined a template specification for LMI for each type of matrices. Which directory? Make a directory called core_algebra"

Reply:
on then maybe add a directory `spectahedra` in `convex_bodies` and put all of them there. @TolisChal  what do you think?